client/signup/steps/user/index.jsx

```diff
@@ -206,7 +206,7 @@ export class UserStep extends Component {
 			subHeaderText = translate( 'Welcome to the WordPress.com community.' );
 		}
 
-		if ( positionInFlow === 0 && flowName === 'onboarding' ) {
+		if ( positionInFlow === 0 ) {
```

Comment:
This will cause the sub-header copy to change in other flows, e.g. signup from akismet.com:

Before:
<img width="574" alt="Screenshot 2021-12-16 at 7 03 30 PM" src="https://user-images.githubusercontent.com/1269602/146396045-e7180eac-a823-4bbf-85b4-9be66b768b5f.png">

After:
<img width="583" alt="Screenshot 2021-12-16 at 7 02 22 PM" src="https://user-images.githubusercontent.com/1269602/146396080-d5ca222d-ba05-4054-9d1e-310d689689ae.png">

I suggest that we remove the line `if ( positionInFlow === 0 && flowName === 'onboarding' ) {` and edit the following line:

```
if ( isReskinned && positionInFlow === 0 ) {
    const loginUrl = login( {
    ...
    ...
```

What do you think?

Reply:
This seems to work. Thank you!
I also had to remove the line right after the condition `subHeaderText = translate( 'First, create your WordPress.com account.' )`